android/src/main/java/com/bitmovin/player/reactnative/PlayerModule.kt

```diff
@@ -428,6 +428,18 @@ class PlayerModule(private val context: ReactApplicationContext) : ReactContextB
         }
     }
 
+    /**
+     * Returns the thumbnail image for the active `Source` at a certain time.
+     * @param nativeId Target player id.
+     * @param time Playback time for the thumbnail.
+     */
+    @ReactMethod
+    fun getThumbnail(nativeId: NativeId, time: Double, promise: Promise) {
+        uiManager()?.addUIBlock {
+            promise.resolve(JsonConverter.fromThumbnail(players[nativeId]?.source?.getThumbnail(time)))
```

Comment:
`Player.getThumbnail` has been deprecated in favour of `Source.getThumbnail` on Android. From my perspective I would be in favour of only wrapping `Source.getThumbnail` in this PR, but I'm not sure what the state is on iOS, just wanted to point this out.

Reply:
@zigavehovec I know about the deprecation on Android but I went for providing both as our `Source` API is not that rich in the React Native SDK yet. Also we don't support playlist, so only one `Source` is available at a time.